### PR TITLE
Escaping * : removed escape \ typo

### LINKED ISSUE #6209: docs: Escaping `*` in argument list no longer necessary
Compare http://docs.micropython.org/en/v1.12/library/pyb.html#pyb.mount and http://docs.micropython.org/en/latest/library/pyb.html#pyb.mount

`latest` is rendered as `\*`, whereas `1.12` is `*`.

An easy fix, but would be good to get to the bottom of why this changed.

### PR DESCRIPTION
**Pull request by agniutkarsh**
closes #6209

### CODE REVIEW

File: docs/library/uos.rst [hunk 1]
```diff
@@ -144,7 +144,7 @@ programs.  Ports that have this functionality provide the :func:`mount` and
 :func:`umount` functions, and possibly various filesystem implementations
 represented by VFS classes.
 
-.. function:: mount(fsobj, mount_point, \*, readonly)
+.. function:: mount(fsobj, mount_point,*, readonly)
```
Comment (dlech) [inline docs/library/uos.rst:147]: don't remove space

File: docs/library/btree.rst [hunk 2]
```diff
@@ -76,7 +76,7 @@ Example::
 Functions
 ---------
 
-.. function:: open(stream, \*, flags=0, pagesize=0, cachesize=0, minkeypage=0)
+.. function:: open(stream,*, flags=0, pagesize=0, cachesize=0, minkeypage=0)
```
Comment (dlech) [inline docs/library/btree.rst:79]: don't remove space

File: docs/library/lcd160cr.rst [hunk 3]
```diff
@@ -37,7 +37,7 @@ For example::
 Constructors
 ------------
 
-.. class:: LCD160CR(connect=None, \*, pwr=None, i2c=None, spi=None, i2c_addr=98)
+.. class:: LCD160CR(connect=None,*, pwr=None, i2c=None, spi=None, i2c_addr=98)
```
Comment (dlech) [inline docs/library/lcd160cr.rst:40]: don't remove space

File: docs/library/machine.Pin.rst [hunk 4]
```diff
@@ -42,7 +42,7 @@ Usage Model::
 Constructors
 ------------
 
-.. class:: Pin(id, mode=-1, pull=-1, \*, value, drive, alt)
+.. class:: Pin(id, mode=-1, pull=-1,*, value, drive, alt)
```
Comment (dlech) [inline docs/library/machine.Pin.rst:45]: don't remove space

File: docs/library/machine.Pin.rst [hunk 5]
```diff
@@ -106,7 +106,7 @@ Constructors
 Methods
 -------
 
-.. method:: Pin.init(mode=-1, pull=-1, \*, value, drive, alt)
+.. method:: Pin.init(mode=-1, pull=-1,*, value, drive, alt)
```
Comment (dlech) [inline docs/library/machine.Pin.rst:109]: don't remove space

File: docs/library/machine.Pin.rst [hunk 6]
```diff
@@ -179,7 +179,7 @@ Methods
 
    Availability: WiPy.
 
-.. method:: Pin.irq(handler=None, trigger=(Pin.IRQ_FALLING | Pin.IRQ_RISING), \*, priority=1, wake=None, hard=False)
+.. method:: Pin.irq(handler=None, trigger=(Pin.IRQ_FALLING | Pin.IRQ_RISING),*, priority=1, wake=None, hard=False)
```
Comment (dlech) [inline docs/library/machine.Pin.rst:182]: don't remove space

File: docs/library/machine.SPI.rst [hunk 7]
```diff
@@ -29,7 +29,7 @@ Constructors
 Methods
 -------
 
-.. method:: SPI.init(baudrate=1000000, \*, polarity=0, phase=0, bits=8, firstbit=SPI.MSB, sck=None, mosi=None, miso=None, pins=(SCK, MOSI, MISO))
+.. method:: SPI.init(baudrate=1000000,*, polarity=0, phase=0, bits=8, firstbit=SPI.MSB, sck=None, mosi=None, miso=None, pins=(SCK, MOSI, MISO))
```
Comment (dlech) [inline docs/library/machine.SPI.rst:32]: don't remove space

File: docs/library/machine.Signal.rst [hunk 8]
```diff
@@ -75,7 +75,7 @@ Constructors
 ------------
 
 .. class:: Signal(pin_obj, invert=False)
-           Signal(pin_arguments..., \*, invert=False)
+           Signal(pin_arguments...,*, invert=False)
```
Comment (dlech) [inline docs/library/machine.Signal.rst:78]: don't remove space

File: docs/library/machine.UART.rst [hunk 9]
```diff
@@ -43,7 +43,7 @@ Constructors
 Methods
 -------
 
-.. method:: UART.init(baudrate=9600, bits=8, parity=None, stop=1, \*, ...)
+.. method:: UART.init(baudrate=9600, bits=8, parity=None, stop=1,*, ...)
```
Comment (dlech) [inline docs/library/machine.UART.rst:46]: don't remove space

**Review by dlech — CHANGES_REQUESTED** [review]
It looks like you have accidentally removed some space characters in a few cases.

**Comment by Zenix27** [comment]
Added Space @dlech @jimmo

**Comment by dpgeorge** [comment]
Thank you for the contribution!  Rebased, squashed and merged in e76c7466b62c940150cf1b5000e18f60a5efa2e3

**Pull request closed without merging**